### ISSUE TITLE
Lag before CLI prompt comes back

### ISSUE THREAD

**Issue by findepi** [prestosql/presto#2008]
Observed when running CLI against local single-node Presto:

```
presto:default> set session hive.collect_column_statistics_on_write = false;
SET SESSION                                                                        // it takes 1s to show that
presto:default>                                                                    // it takes another 2s to show that   
```

**Comment by ryanrupp** [comment]
Should be resolved by #2022 (at least for the second gap noted here). I'm not sure about the first gap, the first execution is about 1s guessing just due to class loading/first initialization of things. Subsequent statements come back quickly but could maybe be snappier? I pulled up an old version from a few months ago and seemed about the same.

**Comment by findepi** [comment]
@ryanrupp thank you for fixing the regression!
it's definitely responsive now. I am not pasting long `SET SESSION` sequences, so the remaining delay doesn't matter to me.

Thanks!
